src/components/SuggestedJourneys/SuggestedJourneys.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { stationsById } from "stations";
+import { JourneyParams } from "types";
+
+import styles from "./SuggestedJourneys.module.scss";
+
+type Props = {
+    journeys: JourneyParams[];
+};
+
+export function getJourneyUrl(journey: JourneyParams) {
+    return `/?from=${journey["fromStationId"]}&to=${journey["toStationId"]}&day=${journey["day"]}&time=${journey["time"]}`;
+}
+
+function getClass(time: JourneyParams) {
```

Comment:
A more descriptive name (`getTimeOfDayClass`) would be helpful here. I would also rename the `time` argument to something reflective of its type (like `params` or `journeyParams`)